### PR TITLE
Merging duped branch

### PR DESCRIPTION
**Pull request by merezal**

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <!DOCTYPE html>
-
+<head>
+    <meta charset="utf-8" />
+    <title>{{title}}</title>
```
Comment (julianfortune) [inline views/layouts/main.handlebars:4]: This looks awesome! 🔥🔥🔥

File: package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
     "scripts": {
         "test": "echo \"Error: no test specified\" ",
         "start": "node server.js",
-        "build": "echo \"Error: no build specified\""
+        "build": "handlebars views/partials/preview.handlebars -f public/previewTemplate.js",
```
Comment (julianfortune) [inline package.json]: Do we need to compile this template?

File: public/index.js [hunk 3]
```diff
@@ -1,11 +1,19 @@
 var search_box = document.querySelector('#nav_search_box');
-search_box.addEventListener('input', search);
+search_box.addEventListener('change', search);
 function search(event){
```
Comment (julianfortune) [inline public/index.js:3]: Can you explain what's going on in here please?

File: public/index.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
-        if((searches[i].name.includes(search_box.value)||(searches[i].description.includes(search_box.value))){
-            
-        }
-    }
+   // searches = require('exampleQuiz.json');
+    //for(var i=0;i<searches.length;i++){
+     //   console.log(searches)
+       // if((searches[i].name.includes(search_box.value))||(searches[i].description.includes(search_box.value))){
+            //FIX ME//
+            var url='/search/'+search_box.value.toLowerCase();
```
Comment (julianfortune) [inline public/index.js]: We're going to need to replace spaces with something else. Could you look into that?

File: views/partials/preview.handlebars [hunk 5]
```diff
@@ -1,5 +1,5 @@
-<div>
+<div class="search_result_preview">
     <h3>{{name}}</h3>
-    <p class="num_count">{{card_count}}</p>
-    <p class="description">{{description}}</p>
+    <p class="num_count">{{card_count}} Questions</p>
+    <p class="description">{{creation_date}}: {{description}}</p>
```
Comment (julianfortune) [inline views/partials/preview.handlebars:4]: This looks good!

**Review by julianfortune — CHANGES_REQUESTED** [review]
Great work Michael, just a couple of questions

File: package.json (hunk 2 above)
Comment (merezal) [reply package.json]: not really we can actually remove that

File: public/index.js (hunk 3 above)
Comment (merezal) [reply public/index.js:3]: Once you press enter with your search it converts the input to lowercase and sends a redirect with a request to '/search/:input'

File: public/index.js (hunk 4 above)
Comment (merezal) [reply public/index.js]: Done

Comment (merezal) [reply public/index.js]: Now it just removes spaces, but we can change that later if we need

File: exampleQuiz.json [hunk 6]
```diff
@@ -1,50 +0,0 @@
-{
-    "_id": "1923",
-    "name": "State Capitals",
-    "description": "The capitals of all 50 states in the U.S.",
-    "creation_date": "1559404138",
-    "tags": [
```
Comment (julianfortune) [inline exampleQuiz.json]: Why did you delete the example quiz?

File: public/index.js (hunk 3 above)
Comment (julianfortune) [reply public/index.js:3]: Sick

File: public/index.js [hunk 7]
```diff
@@ -1,11 +1,7 @@
...
+search_box.addEventListener('change', search);
 function search(event){
-    searches = require('../exampleQuiz.json');
-    for(var i=0;i<searches.length;i++){
-        if((searches[i].name.includes(search_box.value)||(searches[i].description.includes(search_box.value))){
-            
-        }
-    }
-
+    var url='/search/'+search_box.value.replace(/\s+/g, '').toLowerCase();
```
Comment (julianfortune) [inline public/index.js:4]: Can we undo this " "-replacement process? Do we even need to get spaces back on the server? 🤔

**Review by julianfortune — CHANGES_REQUESTED** [review]
What happened to example quiz? And just a quick clarification question about the spaces

File: exampleQuiz.json (hunk 6 above)
Comment (merezal) [reply exampleQuiz.json]: Don't even know how that happened, fixed now

File: public/index.js (hunk 7 above)
Comment (merezal) [reply public/index.js:4]: Can't undo it but we could replace spaces with + or some other special character by replacing between (/\s+/g, '+'). I don't know why we would need spaces, unless you want to search the description for a whole sentence.

Comment (julianfortune) [reply public/index.js:4]: Sounds good! I trust your judgment

**Comment by julianfortune** [comment]
I'm gonna fix merge issue and merge

**Comment by julianfortune** [comment]
Hope I did all that right! 🤞🏼

**Comment by julianfortune** [comment]
Oh shit didn't see you want to go into search. Did you mean to do that?

**Review by julianfortune — APPROVED** [review]
🏆

**Comment by julianfortune** [comment]
hey Michael, i think we should get the handlebars template changes into master. Do you want to just merge this whole thing into master?

**Comment by merezal** [comment]
Sounds good if it looks good we can merge it to master
